aptos-move/aptos-aggregator/src/aggregator_extension.rs

```diff
@@ -279,7 +270,7 @@ impl Aggregator {
         {
             // If value is Unset, we read it
             if let SpeculativeStartValue::Unset = speculative_start_value {
-                if *delta != SignedU128::Positive(0) || !history.is_empty() {
```

Comment:
Earlier, we were comparing delta != 0. Now, we are comparing delta = 0.

Reply:
Nice catch! Unit test caught this, I've just fixed it in the next PR by accident